openapi.yml

```diff
@@ -128,7 +128,6 @@ components:
           format: float
         encoded_date:
           type: string
-          format: date-time
     DroFile:
```

Comment:
Just let any string be returned for this field, rather than validating the response.  This would allow something like this to be returned ok: `2001-09-11T16:03:24`

Reply:
Here an example that works fine: https://argo.stanford.edu/view/druid:bb012xz4244